src/utils/workspaceUtils.ts

```diff
@@ -33,6 +33,17 @@ export async function selectWorkspaceFile(context: IActionContext, placeHolder:
     if (workspace.workspaceFolders?.length === 1) {
         // if there's a fileExtension, then only populate the quickPick menu with that, otherwise show the current folders in the workspace
         const files = globPattern ? await workspace.findFiles(globPattern) : await workspace.findFiles('**/*');
+
```

Comment:
I'd log the counts both at 0 here.

Reply:
It should already be logged before it hits this point I believe

Reply:
Oh nevermind I'm dumb, I see what you mean now hahaha

End of day brain is mush brain

Reply:
<img src="https://media3.giphy.com/media/YC1RQd6Gsp1Cg/giphy.gif"/>